slides/timeline/2019.md

```diff
@@ -98,4 +98,12 @@
 
 ## SDN x Cloud Native Meetup #22
 ###### `2019/11/30`
-共筆連結(Collaborative Writing)： https://hackmd.io/@CNTUG/meetup-22
\ No newline at end of file
+共筆連結(Collaborative Writing)： https://hackmd.io/@CNTUG/meetup-22
+
+| Topic       | Speaker        | Links |
+|-------------|----------------|--------------|
+| How to achieve canary deployment on Kubernetes | John Chen(Grindr) | [Slides]( https://www.slideshare.net/ssuserd7d9ae/how-to-achieve-canary-deployment-on-kubernetes) |
+| Massive Bare-Metal Operating System Provisioning Improvement |  Date Huang(Engineer, Edgecore Networks) | [Slides](https://docs.google.com/presentation/d/10CPDuAd8EfPcuKAZ-f2GYnvsMwTtOzaIl_3MizAJMMA/edit?usp=sharing) |
+| SDN演義 - 初出茅廬 (1/n) | JianHao Chen(SDN 說書人, Edgecore Networks) | [Slides](https://drive.google.com/file/d/1aYUoVzlJi-LgNnFFfWbpMzFtmQpyf3B1/view?usp=sharing) |
+| Lightening Talk: 2019 LINE DevDay 分享 | Kyle Bai | [Slides]() |
```

Comment:
It seems better to use TBA or static text instead of a hyperlink?

Reply:
I agree @pohsienshih. plz, fix it.